project/Build.scala

```diff
@@ -79,7 +83,9 @@ object DottyBuild extends Build {
 
       tuning ::: agentOptions ::: travis_build ::: fullpath
     }
-  )
+  ) ++ (if (isTravisBuild) Seq() else addCommandAlias("test", "partest"))
```

Comment:
Making `test` mean `test` feals clearer to me, as travis could also benefit from it(it would run `test`, not `partest`). 

Silently replacing default task of sbt feels like a very unintuitive action for newcomers, who would naturally expect `test` to run tests in the same way as travis does it. This cryptic detail would make it harder for newcomers to reproduce failures seen on travis.

If's it's indeed impossible or very hard to achieve, we can live with this.